Server/DeviceManagement/DeviceManagement.Application/CQRS/Commands/CreateRouterCommand.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+using ModularHouse.Libraries.InternalMessaging.CQRS.Abstractions.Command;
+using ModularHouse.Server.DeviceManagement.Application.Dto;
+
+namespace ModularHouse.Server.DeviceManagement.Application.CQRS.Commands;
+
+public record CreateRouterCommand(RouterInputDto Router, Guid AreaId, Guid DeviceId, Guid UserId) : ICommand;
```

Comment:
CreatedByUser and LastUpdatedByUser will be set by current UserSession. Must be implemented when Auth is ready. For now add TODOs.
But remove UserId from commands.